opencti-platform/opencti-graphql/src/modules/workspace/workspace-domain.ts

```diff
@@ -135,3 +142,45 @@ export const workspaceEditContext = async (context: AuthContext, user: AuthUser,
   return storeLoadById(context, user, workspaceId, ENTITY_TYPE_WORKSPACE)
     .then((workspaceToReturn) => notify(BUS_TOPICS[ENTITY_TYPE_WORKSPACE].EDIT_TOPIC, workspaceToReturn, user));
 };
+
+function checkDashboardConfigurationImport(parsedData: any) {
```

Comment:
we usually don't declare functions with `function`, more with 
`const myfunction = (args) => { }`